### ISSUE TITLE
Type 'null' is not assignable to type 'IconOrRenderFunction | undefined'

### ISSUE THREAD

**Issue by aslampr07** [wojtekmaj/react-time-picker#296]
When setting  `clearIcon={null}` I am getting the following typescript error

```
Type 'null' is not assignable to type 'IconOrRenderFunction | undefined'.ts(2322)
TimePicker.d.ts(13, 5): The expected type comes from property 'clearIcon' which is declared here on type 'IntrinsicAttributes & Pick<TimePickerProps, "className" | "autoFocus" | "id" | "name" | "onChange" | "onFocus" | "value" | "required" | ... 46 more ... | "secondPlaceholder"> & Pick<...> & Pick<...>'
```

Any idea on how to fix this?
![Screenshot_20230411_134246](https://user-images.githubusercontent.com/19869134/231098297-7524b3e6-0e7e-4633-9ebb-aa5368e10588.png)

**Comment by wojtekmaj** [comment]
My bad

**Comment by wojtekmaj** [comment]
Fixed in v6.0.4

**Comment by aslampr07** [comment]
Thanks @wojtekmaj Search for the ideal timepicker for me ends here.

I was pulling my hair trying to create a custom timepicker for my project.
